source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,105 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
```

Comment:
Done. 

Added *mux* is short for multiplex, because it acts like a multiplexor or manifold,
taking in connections from many collectors, and distributing them to a data
store.